src/bin

```diff
@@ -48,7 +65,10 @@ prog
         } else {
           console.log(kleur.red(`${file} is invalid:`));
           data.errors.forEach((err) => {
-            console.log(kleur.red(`  - ${err}`));
+            console.log(kleur.red(`ERROR - ${err}`));
```

Comment:
Can you please remove the `ERROR` message and keep it as it was before? I would like to avoid breaking changes on the formating :)

Reply:
can do